node/runtime/src/lib.rs

```diff
@@ -80,7 +80,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
 	spec_version: 122,
-	impl_version: 123,
+	impl_version: 124,
```

Comment:
It looks like the runtime behavior is changed by this PR and thus according to the comment above we need to bump `spec_version`.